pkg/misc/other_test.go

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2022 CECTC, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package misc
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/cectc/dbpack/third_party/parser/opcode"
+)
+
+func TestIsFloat64Equal(t *testing.T) {
+	cases := map[string]struct {
+		a, b float64
+		want bool
+	}{
+		"0.0 == 0.0": {0.0, 0.0, true},
+		"0.0 != 1.0": {0.0, 1.0, false},
+		"1.0 == 1.0": {1.0, 1.0, true},
+		"1.0 != 0.0": {1.0, 0.0, false},
+	}
+
+	for name, c := range cases {
+		t.Run(name, func(t *testing.T) {
+			got := IsFloat64Equal(c.a, c.b)
+			if got != c.want {
+				t.Errorf("IsFloat64Equal(%v, %v) = %v, want %v", c.a, c.b, got, c.want)
+			}
+		})
+	}
+}
+func TestIsFloat32Equal(t *testing.T) {
```

Comment:
please add a blank line before this func

Reply:
done